File: python/Kode program/HouseOldItem.py
class HouseOldItem:
    def __init__(self):
        self.item = ""
        self.price = 0.0
        self.room = ""

    def setItem(self, i):
        self.item = i

    def getItem(self):
        return self.item

    def setPrice(self, p):
        self.price = p

    def getPrice(self):
        return self.price

    def setRoom(self, r):
        self.room = r

    def getRoom(self):
        return self.room

    def printItem(self):
        print(f"Item: {self.item}")
        print(f"Price: {self.price}")
        print(f"Room: {self.room}")

File: java/Kode program/HouseOldItem.java
public class HouseOldItem {
    private String item;
    private double price;
    private String room;

    public void setItem(String i) { item = i; }
    public String getItem() { return item; }

    public void setPrice(double p) { price = p; }
    public double getPrice() { return price; }

    public void setRoom(String r) { room = r; }
    public String getRoom() { return room; }

    public void printItem() {
        System.out.println("Item: " + item);
        System.out.println("Price: " + price);
        System.out.println("Room: " + room);
    }
}
